src/Component/BlogCard.css

```diff
@@ -0,0 +1,24 @@
+.blog__card__container{
```

Comment:
Learn about BEM in CSS. naming of the variable that you have used is similar but doesn't follow the convention. It would be easier if you can practice that.